sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs

```diff
@@ -72,6 +78,9 @@ public TextAnalyticsClientOptions(ServiceVersion version = LatestVersion)
         {
             Version = version;
             this.ConfigureLogging();
+
+            //Default Audience to Azure Public Cloud
+            this.Audience ??= TextAnalyticsAudience.AzureResourceManagerPublicCloud;
```

Comment:
```suggestion
```

Reply:
The other default values are defined inline with the property declaration.   We should consider doing the same for Audience.